### PR TITLE
New Instructor Followup Procedures

### PR DESCRIPTION
**Pull request by jiffyclub**
Getting started on the new instructor followup procedures from swcarpentry/board_pvt#145:

> We need a procedure document that describes the who, how, and when of what to do when an instructor completes instructor training. That includes, in particular, how to introduce them to the instructor mailing and other communication channels, how to get them connected to their first workshop, how to help them organize a workshop nearby if none are available, potentially how to pair them with more senior instructors, and, in particular, what emails to send to that instructor to see how they're doing, where they want to teach, whether their institution wants to be a partner, etc.

Note that this creates a new "checklists" directory here in the board repo where we can put these such things (in simple Markdown format), but I'm open to other locations.

Other things I'm thinking of adding are a followup from the mentoring committee lead if the instructor hasn't taught within six months (this would require a notification from AMY) and an email from the executive director about institutional partnership.

### CODE REVIEW

File: checklists/instructor-followup/instructor-trainer.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Immediately after students have completed instructor training please
+send this concluding email:
+
+    Thanks again for taking part in Software Carpentry instructor training
+    - we're pleased to have you on the team.  When you have a minute,
+    please:
+
+    1. Take a moment to go to AMY [1] and update your profile so we know where
```
Comment (rgaiacs) [inline checklists/instructor-followup/instructor-trainer.md]: Can people do that?

File: checklists/instructor-followup/instructor-trainer.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        you are and what you're comfortable teaching - that will help us
+        match you with workshops.
+
+    2. Send me a three-line bio and a photo to add to our website (see
+        our team page [2] for examples).
+
+    3. Make sure you're on the 'instructors' mailing list (signup at [3])
```
Comment (rgaiacs) [inline checklists/instructor-followup/instructor-trainer.md]: I thought that we add new instructors to the 'instructors' mailing list right after they get their badge.

File: checklists/instructor-followup/instructor-trainer.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        so that you'll get notice about upcoming workshops that need
+        instructors - if you'd like us to help you organize one where you
+        are, please give me a shout.
+
+    Please also think about joining the discussion list (signup also at
+    [3]) - it's relatively low traffic, and high signal-to-noise, and it'd
```
Comment (rgaiacs) [inline checklists/instructor-followup/instructor-trainer.md]: Replace [3] with the link to the subscription page of each mailing list.

**Comment by rgaiacs** [comment]
> Other things I'm thinking of adding are a followup from the mentoring committee lead if the instructor hasn't taught within six months (this would require a notification from AMY) and an email from the executive director about institutional partnership.

+1 from me to do that.

**Comment by rgaiacs** [comment]
@ChristinaLK can you add this to discussion by the mentoring subcommittee?

**Comment by ChristinaLK** [comment]
Sorry, didn't get to this today, but will add it to the rotating agenda.

**Comment by raynamharris** [comment]
Hi @ChristinaLK . How does this fit in with the "shepherding" proposal?  Should we add this to our agenda for the next meeting?

**Comment by ChristinaLK** [comment]
It totally fits in.  @jiffyclub , I may have you coordinate with the leaders of the mentoring committee's new proposal for supporting new instructors post-training.

**Comment by ChristinaLK** [comment]
Sorry for dropping the ball on this, btw!  Will try to get this on the agenda for the next meeting.

**Pull request merged**